integrations/saltstack/saltstack-remediation/README.md

```diff
@@ -0,0 +1,86 @@
+## Overview
+
+<!-- Sensu Integration description; supports markdown -->
+
+The SaltStack integration is a [Sensu Event Pipeline][pipeline] that enables SaltStack as an automated remediation provider for Sensu Go. This integration enables users to configure "remediation actions" that trigger [SaltStack Functions][saltstack-functions].
+
+<!-- Provide a high level overview of the integration contents (e.g. checks, filters, mutators, handlers, assets, etc) -->
+
+This integration includes the following resources:
+
+* `saltstack` ([pipeline])
+* `saltstack` ([handler])
+* `sensu/sensu-saltstack-handler:0.4.0` ([asset])
+
+## Dashboards
+
+<!-- List of supported dashboards w/ screenshots (supports png, jpeg, and gif images; relative paths only; e.g. `![](img/dashboard-1.png)` )-->
+
+There are no compatible dashboards for this integration.
+
+## Setup
+
+<!-- Sensu Integration setup instructions, including Sensu agent configuration and external component configuration -->
+<!-- EXAMPLE: what configuration (if any) is required in a third-party service to enable monitoring? -->
+
```

Comment:
It might be good to point directly to
https://bonsai.sensu.io/assets/sensu/sensu-saltstack-handler/versions/0.4.0#annotations
and/or
https://bonsai.sensu.io/assets/sensu/sensu-saltstack-handler/versions/0.4.0#usage-examples
as a reference for what the annotation structure needs to be

Reply:
Link added.